actionpack/lib/action_dispatch/testing/test_process.rb

```diff
@@ -15,9 +15,8 @@ module FixtureFile
       #
       #   post :change_avatar, params: { avatar: fixture_file_upload('files/spongebob.png', 'image/png', :binary) }
       def fixture_file_upload(path, mime_type = nil, binary = false)
-        if self.class.respond_to?(:fixture_path) && self.class.fixture_path &&
-            !File.exist?(path)
-          path = File.join(self.class.fixture_path, path)
+        if self.class.file_fixture_path && !File.exist?(path)
```

Comment:
Can we use `fixture_path` and log a deprecation warning so people know how to fix this for existing relative path code?